crates/kumo-server-common/src/disk_space.rs

```diff
@@ -141,8 +141,8 @@ impl MonitoredPath {
         let info = nix::sys::statvfs::statvfs(&self.path)
             .with_context(|| format!("statvfs({}) failed", self.path.display()))?;
 
-        let blocks_avail = info.blocks_available();
-        let blocks_total = info.blocks();
+        let blocks_avail = info.blocks_available() as u64;
```

Comment:
`fsblkcnt_t` and `fsfilcnt_t` are defined as `u32` (instead of `u64`) on macOS. Widening here seems like easiest fix, and I used `as` since that's already what is being used here. Alternatively, relying on `impl From<u32> for u64` would be slightly safer.